plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -94,14 +94,25 @@ export function getConnectionString(params: PgParsedConnectionParams) {
   return `postgresql://${host}:${port}/${database}`;
 }
 
+function getPort(port: number | undefined): number | undefined {
+  // Port may be NaN as parseInt() is used on the value, passing null will result in NaN being parsed.
+  // https://github.com/brianc/node-postgres/blob/2a8efbee09a284be12748ed3962bc9b816965e36/packages/pg/lib/connection-parameters.js#L66
+  if (port != null && Number.isInteger(port)) {
```

Comment:
nit: I suppose the `port != null &&` could be dropped because `Number.isInteger(null) === false`.

Reply:
ah, yes good point :slightly_smiling_face:

Reply:
applied in [da7b54d](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1982/commits/da7b54d0e3bce706eb396997f52cb199c46207b2)